### PR TITLE
django_check: new module

### PR DESCRIPTION
**Pull request by russoz**
##### SUMMARY

Creates module `django_check`

##### ISSUE TYPE

- New Module/Plugin Pull Request

##### COMPONENT NAME

django_check

### CODE REVIEW

File: plugins/modules/django_check.py [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+__metaclass__ = type
+
+DOCUMENTATION = """
+module: django_check
+author:
+  - Alexei Znamensky (@russoz)
+short_description: Wrapper for C(django-admin check)
+version_added: 9.1.0
+description:
+  - This module is a wrapper for the execution of C(django-admin check).
```
Comment (felixfontein) [inline plugins/modules/django_check.py:17]: How does the module reports the result? Does it fail if the check fails? Maybe add a sentence in the description about that.

**Review by felixfontein — COMMENTED** [review]
Thanks for your contribution!

File: plugins/modules/django_check.py (hunk 1 above)
Comment (russoz) [reply plugins/modules/django_check.py:17]: I will write a bit more, but basically it reports with raw stdout, stderr, rc - no parsing so far. I am happy to add parsing later, right now I am trying to get a couple of  django modules out.

Comment (felixfontein) [reply plugins/modules/django_check.py:17]: Failing (due to rc check) is fine for me, I think it just should be documented :) And parsing can always be added later.

Comment (russoz) [reply plugins/modules/django_check.py:17]: @felixfontein done!

**Review by felixfontein — APPROVED** [review]

**Comment by felixfontein** [comment]
@russoz thanks for your contribution!

**Pull request merged**